defi/src/governance/compound.ts

```diff
@@ -5,7 +5,7 @@ import { updateStats, toHex, getGovernanceSources, getChainNameFromId } from './
 import * as sdk from '@defillama/sdk'
 import { sliceIntoChunks } from '@defillama/sdk/build/util/index'
 import { getProvider } from '@defillama/sdk/build/general'
-import { addICPProposals } from './icp'
+import { addICPProposals } from './nns'
```

Comment:
this file doesnt exist, is the PR complete? maybe you forgot to commit nns.ts?

Reply:
Yes thanks for pointing that out, I renamed the file and forgot to add it afterwards.